assignments/expressionVSstatement.md

```diff
@@ -37,6 +40,7 @@ function add(a = 0, b) {
 }
 add(21);
 ```
+Answer: The program will wait for the value of b to be defined.
```

Comment:
If we don't pass the value for `b` it will default to `undefined` and `21 + undefined` will be `NaN`